src/main/scala/com/typesafe/conductr/client/ConductRController.scala

```diff
@@ -192,6 +210,13 @@ object ConductRController {
         )
       )
     )
+
+  private def extractZipEntry(entryName: String, zipFile: Uri, destDir: File): Uri = {
+    import sbt.NameFilter._
+    val result = IO.unzip(new File(absolute(zipFile).path.toString()), destDir, { fileName: String => fileName.contains(entryName) }).head
+    Uri(result.getAbsolutePath)
+  }
+
```

Comment:
You should do a quick check to see whether the native packager already includes such a function - I'm unsure, but it could be worth looking.